project_euler/problem_26/sol1.py

```diff
@@ -35,8 +53,8 @@ def find_digit(numerator: int, digit: int) -> int:
     return the_digit
 
 
-# Tests
 if __name__ == "__main__":
     import doctest
 
     doctest.testmod()
+    print(solution(digit=int(str(input()).strip())))
```

Comment:
I don't see this in the original file, so please remove this.

Reply:
got it, done